tests/components/vizio/conftest.py

```diff
@@ -3,7 +3,9 @@
 
 import pytest
 from pyvizio.api.apps import AppConfig
-from pyvizio.const import DEVICE_CLASS_SPEAKER, MAX_VOLUME
+from pyvizio.const import MAX_VOLUME
+
+from homeassistant.components.media_player import MediaPlayerDeviceClass
```

Comment:
This change is wrong - the import came from `pyvizio` and should continue to do so.

Reply:
Good spot - agreed, done.

Reply:
Good spot - agreed, done.